src/templates/comment-reply-input.jsx

```diff
@@ -145,12 +145,6 @@ const CommentReplyInput = React.createClass({
                 rowCount: 1,
             });
         }
-
-        // Force scroll the parent window to the top on iPhone
-        // Note: This can be removed if we move away from using an iFrame for the notifications window
-        if (window.parent && hasTouch()) {
```

Comment:
I'd be fine with that. we used to check for the presence of the iframe in other places. ideally if we can accomplish with without knowing I'd prefer that but whatevs.

what is the challenge in not acknowledging the iframe?